src/style.css

```diff
@@ -0,0 +1,134 @@
+html {
+  --main-accent: #060b35;
+  --blue-accent: #00acdc;
+  --secondary: #616070;
+
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 400;
+  color: var(--main-accent);
+}
+
+body {
+  margin: 0;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+.pr-card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  box-sizing: border-box;
+
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.pr-card__img {
```

Comment:
Nobody is allowed to either set or reset margins of block elements according to BEM. I would suggest to create and use wrapper for setting margins.